src/tools/ToolSelectMove.ts

```diff
@@ -0,0 +1,147 @@
+import BaseTool from "./BaseTool";
+import BaseMoveTool from "./BaseMoveTool";
+import { GlobalStore } from "@/store";
+import StuntSheetDot from "@/models/StuntSheetDot";
+import { InsideLasso } from "@/models/util/Lasso";
+
+/**
+ * Enables Selection and Moving.
+ * Selections are additive with the shift key, and toggle with the option key.
```

Comment:
I think the equivalent for option is alt for windows/linux
```suggestion
 * Selections are additive with the shift key, and toggle with the option/alt key.
```

Reply:
yep!

Reply:
done